### PR TITLE
feat(FOROME-1230): implement search bar for the drawer

### PR DESCRIPTION
**Pull request by Lapk1n**
https://quantori.atlassian.net/browse/FOROME-1230

I got some doubts about using `document.querySelectorAll(className)` in `use-scroll-to-item.ts`

### CODE REVIEW

**Comment by AndreyUstyumenko** [comment]
Maybe pick a bright yellow/orange color for the selection?
Screen Shot 2022-07-04 at 22 20 06

File: src/core/hooks/use-scroll-to-item.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  }
+
+  const detectKey = (e: KeyboardEvent) => {
+    if (e.key === 'Enter') {
+      scrollToItem()
+    }
+  }
+
+  useEffect(() => {
+    window.addEventListener('keydown', detectKey)
```
Comment (AndreyUstyumenko) [inline src/core/hooks/use-scroll-to-item.ts]: would've work better if was active only when the searchbar is focused

File: src/pages/ws/ui/variant-drawer/variant-drawer.tsx [hunk 2]
```diff
@@ -36,6 +39,33 @@ export const VariantDrawer = observer(
...
+
+    const handleChangeActiveAspect = (aspect: string) => {
+      refIndex.current = 0
+      setGalleryActiveAspect(aspect)
+    }
+
+    const onChange = (value: string) => {
+      setSearchValue(value)
+
+      if (value) {
```
Comment (AndreyUstyumenko) [inline src/pages/ws/ui/variant-drawer/variant-drawer.tsx]: Maybe if there's been **found** any value

**Comment by AndreyUstyumenko** [comment]
BTW, debatable thing, but maybe display how many entries have been found?

**Comment by Lapk1n** [comment]
Screenshot 2022-07-06 at 10 37 02
Something like this? I'll talk to Anastasia today.

**Review by AndreyUstyumenko — APPROVED** [review]

**Review by EvgeniySobolev — APPROVED** [review]

**Pull request merged**